$NetBSD: patch-config.h.in,v 1.2 2022/11/21 12:00:01 wiz Exp $

Do not unconditionally assume the existence of the glibc-specific strverscmp(3).
See upstream commit 87042fa51da7c14a7600bbf8420105dd91675757

--- config.h.in.orig	2022-08-12 09:32:25.000000000 +0000
+++ config.h.in
@@ -161,7 +161,7 @@
 #define HAVE_STRING_H 1
 
 /* Define to 1 if you have the `strverscmp' function. */
-#define HAVE_STRVERSCMP 1
+#mesondefine HAVE_STRVERSCMP
 
 /* Define to 1 if you have the <sys/stat.h> header file. */
 #define HAVE_SYS_STAT_H 1
